### PR TITLE
count() usage optimization

### PR DESCRIPTION
**Pull request by samsonasik**
- [x] Is this related to quality assurance?

### CODE REVIEW

File: src/Formatter/Simple.php [hunk 1]
```diff
@@ -63,7 +63,7 @@ public function format($event)
 
         $event = parent::format($event);
         foreach ($event as $name => $value) {
-            if ('extra' == $name && count($value)) {
+            if ('extra' == $name && $value) {
```
Comment (michalbundyra) [inline src/Formatter/Simple.php]: Not sure about this change, what if `$value` is `countable` object?

Comment (samsonasik) [reply src/Formatter/Simple.php]: ok, reverted the change

Comment (michalbundyra) [reply src/Formatter/Simple.php]: In that case I would add test on it

File: src/Formatter/Xml.php [hunk 2]
```diff
@@ -66,15 +66,15 @@ public function __construct($options = [])
                 'rootElement' => array_shift($args)
             ];
 
-            if (count($args)) {
+            if ($args) {
```
Comment (Xerkus) [inline src/Formatter/Xml.php]: I would rather change  it to `empty()` as more expressive

Comment (samsonasik) [reply src/Formatter/Xml.php]: updated.

**Comment by Xerkus** [comment]
@samsonasik wow, that was quick!

**Comment by Xerkus** [comment]
Can you also rebase it on top of master?

**Comment by samsonasik** [comment]
rebased against master.

**Comment by Xerkus** [comment]
@samsonasik thank you

**Pull request merged**